marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/QueryStepRunner.java

```diff
@@ -234,7 +231,7 @@ public Job run(Collection uris) {
         c.setClient(stagingClient);
 
         stepStatusListeners.forEach((StepStatusListener listener) -> {
-            listener.onStatusChange(this.jobId, 0, 0, 0,  "running collector");
+            listener.onStatusChange(this.jobId, 0, JobStatus.RUNNING, 0, 0,  "running collector");
```

Comment:
I think 'status' is all mixed up. We can discuss this again on Monday

Reply:
it does not do harm to expose the job status. I think it is more clear...